src/views/PlayView.vue

```diff
@@ -30,65 +44,73 @@ export default {
   data() {
     return {
       showRoomCodeField: true,
-      name: '',
+      name: this.$userStore.name,
       code: ''
-    };
+    }
   },
   created() {
-    if (this.name === '') {
-      this.name = this.$userStore.name;
-    } 
-    this.$clueStore.clear();
-    this.$gameStore.clear();
+    this.$clueStore.clear()
+    this.$gameStore.clear()
+  },
+  computed: {
+    buttonClass() {
+      return (type) => {
+        return type === 'host'
```

Comment:
while this is very clever - it is tricky to read! might be worth using the more boring if else statements for readability.

Reply:
this has been reverted back to if statements for readbility